tensorflow/tools/docker/README.md

```diff
@@ -44,6 +44,10 @@ accomplished via
 
 Alternately, you can use the `docker_run_gpu.sh` script in this directory.
 
+In order to set Jupyter Notebook to require a password, the `-e PASSWORD=pass` option must be provided
+
+    $ docker run -it -p 8888:8888 $CUDA_SO $DEVICES -e PASSWORD=pass gcr.io/tensorflow/tensorflow-devel-gpu
```

Comment:
Users of Jupyter Notebook docker images of TensorFlow typically want to use the non-devel images. Does it make sense to change the image name to "gcr.io/tensorflow/tensorflow-gpu" or "gcr.io/tensorflow/tensorflow"?